src/code/padmgr.c

```diff
@@ -44,132 +56,133 @@ void PadMgr_UnlockReleaseControllerQueue(PadMgr* padmgr, OSMesgQueue* ctrlrqueue
     }
 }
 
-void PadMgr_Lock2(PadMgr* padmgr) {
+void padmgr_LockContData(PadMgr* padmgr) {
```

Comment:
I think Controller is better than Cont even if it is a bit longer. I also see controller is referenced as Pad in other functions. It might be beneficial to keep these consistent